packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -278,28 +307,29 @@ class AndroidCameraCameraX extends CameraPlatform {
   Future<void> _bindPreviewToLifecycle() async {
     assert(processCameraProvider != null);
     assert(cameraSelector != null);
+    assert(preview != null);
 
+    final bool previewIsBound = await processCameraProvider!.isBound(preview!);
```

Comment:
@reidbaker @gmackall @bparrishMines The other major change I made is not relying on class variables to know if a `UseCase` has been bound or not. I wrapped [`isBound`](https://developer.android.com/reference/androidx/camera/lifecycle/ProcessCameraProvider#isBound(androidx.camera.core.UseCase)) to achieve that.